src/app/api/events/byDate/route.ts

```diff
@@ -0,0 +1,37 @@
+import { NextResponse } from 'next/server';
+import { GET as getEvents } from '../route';
+import type { Event } from '@/../lib/types.d.ts';
+
+const calendarId =
+  'c_893d7ec01b0d651ddfadf46f6792b1d470abe97c6d9f33157a1f4be2d4420a51@group.calendar.google.com';
```

Comment:
Is this value meant to be hardcoded into the codebase?

Reply:
Left it in since its technically public info. Didn't really want to add another env variable. Can remove and move to env variable instead if that seems better